C/common/config_category.cpp

```diff
@@ -1463,17 +1463,6 @@ ostringstream convert;
 		{
 			convert << ", \"file\" : \"" << m_file << "\"";
 		}
-		if (m_options.size() > 0)
-		{
-			convert << ", \"options\" : [ ";
-			for (int i = 0; i < m_options.size(); i++)
-			{
-				if (i > 0)
-					convert << ",";
-				convert << "\"" << m_options[i] << "\"";
-			}
-			convert << "]";
-		}
```

Comment:
Is this a repetitive code?

Actual handling of options having enumeration type is taking care with line https://github.com/fledge-iot/fledge/blob/557d47a38570017f2be4f101be97701a4d04bba5/C/common/config_category.cpp#L1386

Don't we have a coverage in unit test?

Reply:
I suspect it was a copy paste error at some point. No, there is no specific unit test for this